unabridged-works-volume-3/original/12-jews-natural-spiritual.adoc

```diff
@@ -795,12 +827,13 @@ and we may feed on nothing but life, and grow in nothing but life and truth,
 to the great glory of our heavenly Father, and to the great joy of our hearts.
 Amen.
 
-=== A FEW WORDS TO ENGLAND, MY NATIVE COUNTRY
+=== A Few Words to England, My Native Country
 
+[.heading-continuation-blurb]
```

Comment:
Wasn't quite sure what I should do with this?

Reply:
Ergh... I'm not sure either. I wonder if maybe a `.saluation` would be better here... 🤔